graphrag/index/operations/chunk_text/chunk_text.py

```diff
@@ -12,20 +12,22 @@
     progress_ticker,
 )
 
+from graphrag.config.models.chunking_config import ChunkingConfig, ChunkStrategyType
```

Comment:
Minor suggestion. Should this be named `ChunkConfig` so the tense matches the other chunk-related models?

Reply:
I like that idea, will review

Reply:
Ok, looking closer it seems most of our config does have the active tense ("embed_graph", "cluster_graph", etc.). I'll leave it for now but we can revist.